### PR TITLE
#162652603 Users should be able to sign out

### PR DESCRIPTION
**Pull request by johnkegz**
#### What does this PR do?
Compare sign-out-162652603 with Develop
#### Description of Task to be completed?
User should be able to sign in when he or she is not using the account.
#### How should this be manually tested?
Put the app Heroku link in the browser, click on sign in and sign in by entering your details, after signing in you can sign out by clicking the sign out button.
#### Any background context you want to provide?
Users should be able to sign out of the app.
#### What are the relevant pivotal tracker stories?
#162652603
#### Screenshots (if appropriate)
screen shot 2018-12-17 at 17 37 31
screen shot 2018-12-17 at 17 37 50
screen shot 2018-12-17 at 17 38 05

### CODE REVIEW

**Comment by shakirandagire** [comment]
@johnkegz Could you remove the files that are not related to logout

File: src/components/ProfileImage.jsx [hunk 1]
```diff
@@ -1,29 +1,105 @@
...
 export class ProfileImage extends Component {
+  state = {
+    username: localStorage.getItem("username"),
+    image:
+      "https://res.cloudinary.com/dksxmwjqs/image/upload/v1545036678/ux5uy7dcbqpzu6hbsjku.jpg"
+  };
   componentDidMount() {
     this.props.fetchProfile();
   }
+  signOuted() {
```
Comment (shakirandagire) [inline src/components/ProfileImage.jsx]: could you make your methods more descriptive and also correct the grammatical error for the function

**Comment by johnkegz** [comment]
> @johnkegz Could you remove the files that are not related to logout

@shakirandagire I have removed files that are not for sign out.

**Review by cdvx — APPROVED** [review]
Change photos to reflect the implemented feature itself,  otherwise,
Looks good 👍🏾

**Comment by johnkegz** [comment]
@cdvx I have changed the pics to the current functionality.

File: src/components/ProfileImage.jsx [hunk 2]
```diff
@@ -1,29 +1,105 @@
...
 import PropTypes from "prop-types";
+import { NavLink } from "react-router-dom";
+
+import { fetchProfile } from "../actions/profileAction";
+
 export class ProfileImage extends Component {
+  state = {
+    username: localStorage.getItem("username"),
+    image:
+      "https://res.cloudinary.com/dksxmwjqs/image/upload/v1545036678/ux5uy7dcbqpzu6hbsjku.jpg"
```
Comment (joshNic) [inline src/components/ProfileImage.jsx:12]: @johnkegz this is a good implementation because it enables users with no profile image to have a default image

**Review by joshNic — APPROVED** [review]
LGTM

File: src/components/ProfileImage.jsx (hunk 2 above)
Comment (johnkegz) [reply src/components/ProfileImage.jsx:12]: Thanks @joshNic

**Comment by emukungu** [comment]
@johnkegz , it's good that you customized the header to a specific page display.
Good work!

File: src/components/ProfileImage.jsx (hunk 1 above)
Comment (johnkegz) [reply src/components/ProfileImage.jsx]: @shakirandagire I have changed the name of the function

**Pull request merged**